Database/Corrections/SeasonOfDiscovery.lua

```diff
@@ -268,6 +273,11 @@ local questsToBlacklistBySoDPhase = {
         [90135] = true, -- Hiding Rogue Quick Draw Teldrassil for now as there are too many icons
         [90136] = true, -- Hiding Rogue Quick Draw Durotar for now as there are too many icons
         [90137] = true, -- Hiding Rogue Quick Draw Tirisfal Glades for now as there are too many icons
+        [90139] = true, -- Hiding Warrior Quick Strike Loch Modan for now as there are too many icons
+        [90140] = true, -- Hiding Warrior Quick Strike Elwynn Forest for now as there are too many icons
+        [90141] = true, -- Hiding Warrior Quick Strike Darkshore for now as there are too many icons
```

Comment:
If we strip the spawns of https://www.wowhead.com/classic/npc=210482/paxnozz to a single one (this NPC has a fixed spawn at `47.8, 16.0`), we can actually show this rune. The other spawns are all added to wowhead, because that NPC swims around, but we can clean that up at a later point.

Reply:
Although he has a fixed spawn point, Paxnozz does swim around the island. Would this affect it at all?

Reply:
Honestly: Doesn't matter at all at this point.

Reply:
```suggestion
```